compiler/src/tokeniser.rs

```diff
@@ -215,4 +216,5 @@ pub enum Token {
 	Plus,
 	EqualsSign,
 	Semicolon,
+	UpToken,
```

Comment:
This is very minor and not worth worrying about, but I feel like this should be "caret" instead of "UpToken" because uptoken implies behaviour but the tokeniser shouldn't really be concerned with behaviour.

That being said, I have definitely done some weird naming for the tokens so I'm not worried (e.g. "OpenAngledBracket" doesn't make sense and should be just "LessThanSign" or something).